### PR TITLE
BRD - Feat: initial ui layout

### PR DESCRIPTION
**Pull request by deetz99**
issue: bcGov/entity#22545

- initial page layout, does not include functionality
- h1 and subtitle
- help text with nuxt content
- 'get started' button - link to nr
- search input + radios
- affiliations table, table header slots with filters
- several enums/interfaces to be used for affiliations query, currently not being used anywhere

Screenshot 2024-07-30 at 5 22 09 PM
Screenshot 2024-07-30 at 5 22 01 PM
Screenshot 2024-07-30 at 5 21 29 PM
Screenshot 2024-07-30 at 5 21 16 PM

### CODE REVIEW

File: business-registry-dashboard/app/app.config.ts [hunk 1]
```diff
@@ -53,10 +53,12 @@ export default defineAppConfig({
         }
       },
       variant: {
+        table_header_clear: 'text-bcGovColor-darkGray rounded-none rounded-tr-md bg-gray-100 hover:bg-gray-200 font-semibold dark:text-white dark:hover:bg-blue-100/10 disabled:bg-{color}-500 dark:disabled:bg-{color}-400 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-{color}-500 dark:focus-visible:outline-white transition-colors duration-300 ease-in-out border-gray-700 focus:border-primary-500 bg-gray-100 hover:bg-gray-200 border-b-[1px] focus:border-b-2 focus:ring-0',
```
Comment (deetz99) [inline business-registry-dashboard/app/app.config.ts:56]: custom button variant for 'clear filter' button in table header slots

File: business-registry-dashboard/app/app.config.ts [hunk 2]
```diff
@@ -53,10 +53,12 @@ export default defineAppConfig({
...
         solid: 'shadow-sm text-white dark:text-gray-900 bg-{color}-500 hover:bg-{color}-600 disabled:bg-{color}-500 dark:bg-[#E0E7ED] dark:hover:bg-bcGovGray-500 dark:disabled:bg-{color}-400 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-{color}-500 dark:focus-visible:outline-white',
-        outline: 'ring-1 ring-inset ring-current text-{color}-500 dark:text-[#E0E7ED] hover:bg-{color}-50 disabled:bg-transparent dark:hover:bg-[#E0E7ED]/25 dark:disabled:bg-transparent focus-visible:ring-2 focus-visible:ring-{color}-500 dark:focus-visible:ring-white'
+        outline: 'ring-1 ring-inset ring-current text-{color}-500 dark:text-[#E0E7ED] hover:bg-{color}-50 disabled:bg-transparent dark:hover:bg-[#E0E7ED]/25 dark:disabled:bg-transparent focus-visible:ring-2 focus-visible:ring-{color}-500 dark:focus-visible:ring-white',
+        select_menu_trigger: 'border-gray-700 focus-visible:border-primary-500 bg-gray-100 hover:bg-gray-200 border-b-[1px] focus-visible:border-b-2 focus:ring-0 h-[42px] rounded-t-m rounded-b-none'
```
Comment (deetz99) [inline business-registry-dashboard/app/app.config.ts:61]: custom button variant for USelectMenu trigger

File: business-registry-dashboard/app/components/HelpTextSection.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/HelpTextSection.vue:1]: component to display 'Help with Starting and Managing a Business' button and help text from nuxt content

File: business-registry-dashboard/app/components/Sbc/PageSection/Card.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/Sbc/PageSection/Card.vue:1]: everything in here that is not the table, so the card wrapper and header items

Screenshot 2024-07-30 at 5 34 37 PM

File: business-registry-dashboard/app/components/Table/Column/Header.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/Table/Column/Header.vue:1]: component to use inside table header slots, slot accepts inputs etc

File: business-registry-dashboard/app/components/content/ProseA.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/content/ProseA.vue:1]: override prosea to add colours and icon if target=_blank

File: business-registry-dashboard/app/composables/useAffiliations.ts
Comment (deetz99) [inline business-registry-dashboard/app/composables/useAffiliations.ts:1]: will need to dissect this, currently not in use

File: business-registry-dashboard/app/pages/index.vue
Comment (deetz99) [inline business-registry-dashboard/app/pages/index.vue:1]: main layout, will need to refactor as functionality is added in

File: business-registry-dashboard/app/stores/affiliations.ts
Comment (deetz99) [inline business-registry-dashboard/app/stores/affiliations.ts:1]: store to handle affiliations, not currently used anywhere, safe to ignore

File: business-registry-dashboard/app/utils/affiliation-invite-info.ts
Comment (deetz99) [inline business-registry-dashboard/app/utils/affiliation-invite-info.ts:1]: utils pulled form auth-web, may or may not end up using, not currently used anywhere

File: business-registry-dashboard/app/utils/mapRequestActionCdToNrType.ts
Comment (deetz99) [inline business-registry-dashboard/app/utils/mapRequestActionCdToNrType.ts:1]: utils pulled form auth-web, may or may not end up using, not currently used anywhere

File: business-registry-dashboard/app/utils/mapRequestTypeCdToNrType.ts
Comment (deetz99) [inline business-registry-dashboard/app/utils/mapRequestTypeCdToNrType.ts:1]: utils pulled form auth-web, may or may not end up using, not currently used anywhere

File: business-registry-dashboard/content/en-CA/start-manage-business-help-text.md
Comment (deetz99) [inline business-registry-dashboard/content/en-CA/start-manage-business-help-text.md:1]: content for help text

File: business-registry-dashboard/content/fr-CA/start-manage-business-help-text.md
Comment (deetz99) [inline business-registry-dashboard/content/fr-CA/start-manage-business-help-text.md:1]: content for help text, will need someone to translate

File: business-registry-dashboard/nuxt.config.ts
Comment (deetz99) [inline business-registry-dashboard/nuxt.config.ts:1]: - ssr for content to render
- add nrUrl for 'get started with a bc based business' button
- content config to handle locales

File: business-registry-dashboard/package.json
Comment (deetz99) [inline business-registry-dashboard/package.json:1]: add package for shared types/enums

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**